pkg/infra/logging.go

```diff
@@ -0,0 +1,29 @@
+package infra
+
+import (
+	"os"
+	"time"
+
+	"github.com/eleanorhealth/go-common/pkg/env"
+	"github.com/rs/zerolog"
+)
+
+func Logger() zerolog.Logger {
+	// Configure Zerolog to work with Google Cloud Logging.
+	// https://github.com/rs/zerolog/issues/174
+	zerolog.TimestampFieldName = "timestamp"
+	zerolog.TimeFieldFormat = time.RFC3339Nano
+	zerolog.LevelFieldName = "severity"
+
+	logger := zerolog.New(os.Stdout)
+
+	if env.Get("ZEROLOG_CONSOLE_WRITER", false) {
+		logger = logger.Output(zerolog.NewConsoleWriter())
+	}
+
+	return logger
+}
+
+func ConsoleLogger() zerolog.Logger {
```

Comment:
Curious why we don't call this from https://github.com/eleanorhealth/go-common/pull/20/files#diff-42a4d310ed71ef086d2861e9a5577190b85252defed66d9cc05cdf701151d1daR21 above and/or if it will get used if we're already returning a console logger where relevant in `Logger()`?

Reply:
We can change that!